### ISSUE TITLE
Error when collecting Laravel notifications

### ISSUE THREAD

**Issue by abalozz** [itsgoingd/clockwork#457]
When I send a message to Slack, I get the error:
```
ErrorException : Undefined property: Illuminate\Notifications\Messages\SlackMessage::$subject
 /var/www/html/current/vendor/itsgoingd/clockwork/Clockwork/DataSource/LaravelNotificationsDataSource.php:192
 /var/www/html/current/vendor/itsgoingd/clockwork/Clockwork/DataSource/LaravelNotificationsDataSource.php:157
 /var/www/html/current/vendor/itsgoingd/clockwork/Clockwork/DataSource/LaravelNotificationsDataSource.php:98
 /var/www/html/current/vendor/itsgoingd/clockwork/Clockwork/DataSource/LaravelNotificationsDataSource.php:49
 /var/www/html/current/vendor/laravel/framework/src/Illuminate/Events/Dispatcher.php:381
...
```

I look into [SlackMessage](https://github.com/laravel/slack-notification-channel/blob/2.0/src/Messages/SlackMessage.php) class, and the property `subject` does not exist. It appears that `subject` property only exists in [MailMessage](https://github.com/laravel/framework/blob/8.x/src/Illuminate/Notifications/Messages/MailMessage.php) (extended from [SimpleMessage](https://github.com/laravel/framework/blob/8.x/src/Illuminate/Notifications/Messages/SimpleMessage.php)).

Maybe in LaravelNotificationsDataSource the method
```php
	protected function resolveSlackChannelSpecific($event, $message)
	{
		return [
			'subject' => $message->subject ?: get_class($event->notification),
			'from'    => $message->username,
			'to'      => $message->channel,
			'content' => $message->content
		];
	}
```
only needs the get_class call?
```php
'subject' => get_class($event->notification),
```

**Comment by itsgoingd** [comment]
Will be fixed in 5.0.5 (https://github.com/itsgoingd/clockwork/commit/697a83a9f7f3d9ceafdc1537380f991414323854), thanks for reporting!

**Comment by itsgoingd** [comment]
Released now.
